graphics/diff-pdf/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+github.setup        vslavik diff-pdf 0.3 v
+github.tarball_from releases
+
+categories          graphics
+maintainers         {@pietvo vanoostrum.org:piet openmaintainer}
```

Comment:
the ```openmaintainer``` should be outside of the curly-brackets as in: 
```maintainers         {@pietvo vanoostrum.org:piet} openmaintainer```

Reply:
Good catch Renee!